alert/views.py

```diff
@@ -233,3 +237,12 @@ def send_json(style, msg):
         return do_register(course_code, email_address, phone, SOURCE_API, send_json, key)
     else:
         raise Http404('Only POST requests are accepted.')
+
+
+class RegistrationViewSet(AutoPrefetchViewSetMixin, viewsets.ModelViewSet):
```

Comment:
If we're planning on replacing the `submit` route with a `POST` request to this view set, then there should be a check to make sure that the `user` field corresponds to the current logged in user. Otherwise, throw a 403 error.